database/SQL/helpers.js

```diff
@@ -1,3 +1,117 @@
 const db = require('./index.js');
 
-module.exports = {};
\ No newline at end of file
+var getProductList = async (count, page) => {
+	return new Promise (async (resolve, reject) => {
+		var result = [];
+		var start = ((page-1) * count) + 1;
```

Comment:
Perhaps the LIMIT and OFFSET sql commands could help you accomplish this purely with sql